src/actions/Entities/VaccinePrescriptionActions.js

```diff
@@ -55,7 +83,7 @@ const reset = () => ({
 
 const selectVaccine = vaccine => ({
   type: VACCINE_PRESCRIPTION_ACTIONS.SELECT_VACCINE,
-  payload: { vaccine },
+  payload: { vaccine, batch: getRecommendedBatch() },
```

Comment:
the call to `getRecommendedBatch` is using the default vaccine, but should be using the one which is passed in here: `vaccine`

Reply:
13b1a42 ha, those last minute untested changes aye